openwisp_controller/geo/admin.py

```diff
@@ -130,14 +131,36 @@ class DeviceLocationFilter(admin.SimpleListFilter):
     title = _("has geographic position set?")
     parameter_name = "with_geo"
 
+    def __init__(self, request, params, model, model_admin):
+        super().__init__(request, params, model, model_admin)
+        if config_app_settings.WHOIS_CONFIGURED:
+            self.title = _("geographic position")
+
     def lookups(self, request, model_admin):
+        if config_app_settings.WHOIS_CONFIGURED:
+            return (
+                ("outdoor", _("Outdoor (Excluding Estimated)")),
```

Comment:
I am thinking if we should mention `Excluding Estimated` here or can rephrase it. @codesankalp what are your thoughts?